tcr-engine/events/test_run_parser.go

```diff
@@ -0,0 +1,46 @@
+package events
+
+import (
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+const regexBuildOutput = `Tests run: [0-9]+, Failures: [0-9]+, Errors: [0-9]+, Skipped: [0-9]+[\r\n]+`
```

Comment:
Comment similar to another comment in engine package: this is maven-specific, so very likely it would be better to have all the extraction from test output string to test run info structure moved to toolchain package